package.json

```diff
@@ -1,148 +1,171 @@
-    "framer-motion": "^6.2.3",
-    "jwks-rsa": "^2.0.4",
-    "leaflet": "^1.9.3",
-    "msw": "1.3.1",
-    "nest-commander": "^3.7.0",
-    "passport": "^0.6.0",
-    "passport-jwt": "^4.0.1",
-    "qrcode": "^1.5.3",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-is": "18.0.0",
-    "react-leaflet": "^4.1.0",
-    "react-redux": "8.0.5",
-    "react-router-dom": "6.4.3",
-    "redux": "^4.2.0",
-    "reflect-metadata": "^0.1.13",
-    "regenerator-runtime": "0.13.7",
-    "rxjs": "^7.1.0",
-    "styled-components": "5.3.6",
-    "swagger-ui-express": "^4.3.0",
-    "tslib": "^2.3.0",
-    "uuid": "^8.3.2",
-    "webpack": "^5.76.0",
-    "zone.js": "~0.11.4"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.23.0",
-    "@babel/preset-react": "^7.14.5",
-    "@cypress/webpack-dev-server": "^2.0.0",
-    "@nestjs/schematics": "^9.0.0",
-    "@nestjs/testing": "^9.2.1",
-    "@nrwl/cli": "15.8.7",
-    "@nrwl/cypress": "15.8.7",
-    "@nrwl/eslint-plugin-nx": "15.8.7",
-    "@nrwl/jest": "15.8.7",
-    "@nrwl/linter": "15.8.7",
-    "@nrwl/nest": "15.8.7",
-    "@nrwl/node": "15.8.7",
-    "@nrwl/nx-cloud": "15.2.3",
-    "@nrwl/react": "15.8.7",
-    "@nrwl/web": "15.8.7",
-    "@nrwl/webpack": "15.8.7",
-    "@nrwl/workspace": "15.8.7",
-    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
-    "@svgr/webpack": "^6.1.2",
-    "@testing-library/cypress": "10.0.1",
-    "@testing-library/react": "^13.4.0",
-    "@types/jest": "29.4.4",
-    "@types/leaflet": "^1.9.0",
-    "@types/node": "16.11.7",
-    "@types/passport-jwt": "^3.0.6",
-    "@types/react": "18.0.25",
-    "@types/react-dom": "18.0.9",
-    "@types/styled-components": "5.1.26",
-    "@types/supertest": "^2.0.11",
-    "@types/validator": "^13.7.10",
-    "@typescript-eslint/eslint-plugin": "5.48.0",
-    "@typescript-eslint/parser": "5.48.0",
-    "babel-jest": "29.4.3",
-    "concurrently": "^6.2.0",
-    "copy-webpack-plugin": "^11.0.0",
-    "cross-env": "^7.0.3",
-    "css-loader": "^6.4.0",
-    "cypress": "^12.8.1",
-    "dotenv": "10.0.0",
-    "eslint": "8.15.0",
-    "eslint-config-prettier": "8.1.0",
-    "eslint-plugin-cypress": "^2.12.1",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-jsx-a11y": "6.6.1",
-    "eslint-plugin-react": "7.31.11",
-    "eslint-plugin-react-hooks": "4.6.0",
-    "html-webpack-plugin": "^5.5.0",
-    "jest": "29.4.3",
-    "jest-environment-jsdom": "29.4.3",
-    "nest-commander-testing": "^3.1.0",
-    "node-fetch": "^3.2.10",
-    "nx": "15.8.7",
-    "prettier": "2.7.1",
-    "react-refresh": "^0.10.0",
-    "react-test-renderer": "18.2.0",
-    "style-loader": "^3.3.0",
-    "stylus": "^0.55.0",
-    "stylus-loader": "^7.1.0",
-    "supertest": "^6.2.2",
-    "ts-jest": "29.0.5",
-    "ts-node": "10.9.1",
-    "typescript": "4.9.5",
-    "url-loader": "^4.1.1",
-    "webpack-merge": "^5.8.0",
-    "webpack-node-externals": "^3.0.0",
-    "whatwg-fetch": "^3.6.2"
-  }
+    "name": "coscrad",
```

Comment:
The diff looks like its only an indent? @aaron-plahn

Reply:
Ah. Yea. I'll have to look at this more carefully before you merge down.